integrated_channels/integrated_channel/tasks.py

```diff
@@ -185,48 +208,40 @@ def transmit_single_subsection_learner_data(username, course_run_id, subsection_
                 subsection_id=subsection_id
             )
 
-        duration = time.time() - start
-        LOGGER.info(
-            '[Integrated Channel] Single learner data transmission task finished.'
-            'Customer: {enterprise_uuid}, Course: {course_run}, Duration: {duration}, Username: {username}'.format(
-                username=username,
-                course_run=course_run_id,
-                duration=duration,
-                enterprise_uuid=enterprise_customer_uuid
+            duration = time.time() - start
+            LOGGER.info(generate_formatted_log(
+                None,
+                enterprise_customer_uuid,
+                user.id,
+                course_run_id,
+                'transmit_single_subsection_learner_data for channels {channels} and for Subsection_id: '
+                '{subsection_id} finished in {duration}s.'.format(
+                        channels=[c.channel_code() for c in enterprise_integrated_channels],
+                        subsection_id=subsection_id,
+                        duration=duration)
             ))
 
 
 @shared_task
 @set_code_owner_attribute
-def transmit_subsection_learner_data(username, channel_code, channel_pk):
+def transmit_subsection_learner_data(job_username, channel_code, channel_pk):
     """
     Task to send assessment level learner data to a linked integrated channel.
 
     Arguments:
-        username (str): The username of the User to be used for making API requests for learner data.
+        job_username (str): The username of the User making API requests for learner data.
```

Comment:
That's a great clarification 👍 has caused me headaches even being familiar with the processes

Reply:
My rule is to NEVER use 'user' alone: roles are better: 'learner_user', 'admin_user', 'api_user' etc